sdk/formrecognizer/Azure.AI.FormRecognizer/perf/FormRecognizerTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Test.Perf;
+
+namespace Azure.AI.FormRecognizer.Perf
+{
+    public abstract class FormRecognizerTest<TOptions> : PerfTest<TOptions> where TOptions : PerfOptions
```

Comment:
A general parent class that all tests will inherit from. It works pretty much as the `FormRecognizerLiveTestEnvironment` in the test project.